app/src/main/java/com/codesoom/assignment/mapper/TaskMapper.java

```diff
@@ -0,0 +1,49 @@
+package com.codesoom.assignment.mapper;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Task를 매핑하는 기능을 가진 클래스입니다.
+ */
+public class TaskMapper {
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    /**
+     * 요청된 본문을 HashMap으로 변환해서 리턴합니다.
+     *
+     * @param request 요청된 본문
+     * @return 변환된 HashMap 리턴
+     * @throws JsonProcessingException Json 변환에 문제가 발생할 경우 던집니다.
+     */
+    public static HashMap<String, String> getRequestMap(String request) throws JsonProcessingException {
+        TypeReference<HashMap<String, String>> hashMapTypeReference = new TypeReference<>(){};
+        return objectMapper.readValue(request, hashMapTypeReference);
```

Comment:
제가 원하는 것이었어요! 정말 좋습니다!
변수명을 지정해서 곧바로 사라지네요 인라인으로 만들어도 좋겠어요!
실무에서도 인라인으로 많이 사용해요!
```suggestion
        return objectMapper.readValue(request, new TypeReference<HashMap<String, String>>(){});
```

여기서 우리는 더 나아가야합니다.
이것이 왜 가능한 것인가! 
우리는 알아야합니다!

정리해서 공유해주세요!